TUT/STA130F24_TUT06_Oct25.ipynb

```diff
@@ -0,0 +1,12137 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "from sklearn.linear_model import LinearRegression\n",
+    "np.random.seed(0)\n",
+    "X = np.random.rand(100, 1) * 10  \n",
+    "y = 2 * X + np.random.randn(100, 1) * 2  \n",
+    "model = LinearRegression()\n",
+    "model.fit(X, y)\n",
+    "intercept = model.intercept_[0]\n",
+    "slope = model.coef_[0][0]\n",
+    "plt.scatter(X, y, color='red', label='Data')\n",
+    "plt.plot(X, slope * X + intercept, color='black', label='Trendline')\n",
+    "plt.title('Example of Fitted Simple Linear Regression Model')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de2190a9",
+   "metadata": {},
+   "source": [
+    "## Tutorial Activity [15 minutes]\n",
+    "\n",
+    "- Break into 3-4 groups and look at the two graphs given below, comparing & contrasting them with respect to their correlation and how well each of them satisfies the 4 assumptions of our simple linear regression model.\n",
+    "<br><br>\n",
+    "- Discuss for 5 minutes and then each group presents for 3 minutes.\n",
+    "\n",
+    "### The Broadway dataset"
```

Comment:
This is an interesting concept... a good idea; but, ...I'm not sure that it's going to be within scope... let me put a pin in this and consider a little bit later where it might fit. 
- I'm concerned about content creed / material bloat 
- **What is needed at this position as of now (instead) is an example of fitting SLR (using the "formula" version of `statsmodels` (not the "X design matrix" approach of the standard base `statsmodels`) and then discussing H0 and the p-value and interpreting all of this**
    - There should probably be 2 examples using whatever data set you go with... one significant, and one not
    - The objective of this Oct 25 TUT is to review hypothesis testing and extend it to SLR